new-site/src/components/Seo.js

```diff
@@ -35,7 +36,77 @@ function Seo({ lang, meta, title, description, pageTitle }) {
           content: `website`,
         },
       ].concat(meta)}
-    />
+    >
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="192x192"
+        href={withPrefix("/favicon/transparent_light_favicon_package/android-chrome-192x192.png")}
+        media="(prefers-color-scheme: light)"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="256x256"
+        href={withPrefix("/favicon/transparent_light_favicon_package/android-chrome-256x256.png")}
+        media="(prefers-color-scheme: light)"
+      />
+      <link
+        rel="apple-touch-icon"
+        href={withPrefix("/favicon/transparent_light_favicon_package/apple-touch-icon.png")}
+        media="(prefers-color-scheme: light)"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="32x32"
+        href={withPrefix("/favicon/transparent_light_favicon_package/favicon-32x32.png")}
+        media="(prefers-color-scheme: light)"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="16x16"
+        href={withPrefix("/favicon/transparent_light_favicon_package/favicon-16x16.png")}
+        media="(prefers-color-scheme: light)"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="192x192"
+        href={withPrefix("/favicon/transparent_dark_favicon_package/android-chrome-192x192.png")}
+        media="(prefers-color-scheme: dark)"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="256x256"
+        href={withPrefix("/favicon/transparent_dark_favicon_package/android-chrome-256x256.png")}
+        media="(prefers-color-scheme: dark)"
+      />
+      <link
+        rel="apple-touch-icon"
+        href={withPrefix("/favicon/transparent_dark_favicon_package/apple-touch-icon.png")}
+        media="(prefers-color-scheme: dark)"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="32x32"
+        href={withPrefix("/favicon/transparent_dark_favicon_package/favicon-32x32.png")}
+        media="(prefers-color-scheme: dark)"
+      />
+      <link
+        rel="icon"
+        type="image/png"
+        sizes="16x16"
+        href={withPrefix("/favicon/transparent_dark_favicon_package/favicon-16x16.png")}
+        media="(prefers-color-scheme: dark)"
+      />
+      <meta name="msapplication-TileColor" content="#da532c" />
+      <meta name="msapplication-TileImage" content="/favicon/transparent_light_favicon_package/mstile-150x150.png" />
```

Comment:
Forgot to add withPrefix here, let's fix it:

- [ ] add withPrefix to meta content also